test-case/check-suspend-resume-with-audio.sh

```diff
@@ -134,6 +136,10 @@ do
         ps --ppid $$ -f
         exit 1
     }
+    # remove -p $process_id option, Need to unset twice
+    unset "opt_arr[${#opt_arr[@]}-1]"
+    unset "opt_arr[${#opt_arr[@]}-1]"
```

Comment:
why twice?

Reply:
1. -p 2. PID

Reply:
You don't need this if you simply use a copy, see `suspend_opts` above.

Every time you change a variable in place, consider making a copy instead. It's always better for correctness and concurrency (it's the default in functional languages)